webpack.config.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+
+module.exports = {
+  entry: './src/client/htmlpugconvert.js',
+  output: {
+    filename: 'main.js',
```

Comment:
I'd name this file something more descriptive than `main.js`, like `pug2html.js` or something like that.